react/LoadMore/index.jsx

```diff
@@ -3,6 +3,8 @@ import PropTypes from 'prop-types'
 import Button from '../Button'
 import Spinner from '../Spinner'
 
+require('intersection-observer') // polyfill for safari (mobile and desktop)
```

Comment:
Here intersection-observer would not be tree shaken since require is used. This means that even if you do not use LoadMore, you would have the intersection observer loaded.

1. We could use `import` here (need to put in the sideEffects config that intersection-observer should not be tree shaken since it has side effects)
2. We could let the app import the `intersection-observer` and warn if the relevant APIs are not present

I think I'd prefer the import solution as it is the most transparent for users.

Reply:
Nice catch!